packages/user-mgnt/src/config/routes.ts

```diff
@@ -278,6 +281,19 @@ export const getRoutes = () => {
         }
       }
     },
+    {
+      method: 'POST',
```

Comment:
Wouldn't GET be more suitable for this? The URL could also be something like `/users/:userId/avatar`. I know it's not like the other URLs here, but if we want to follow REST schema, then it's the direction we should take

Reply:
I liked your suggestion and changed the direction and method to GET